### PR TITLE
Fix spelling errors

### PR DESCRIPTION
**Pull request by chairclr**

### CODE REVIEW

**Comment by chairclr** [comment]
i dont think its ready to be merged, i might need a second opinion

**Comment by chairclr** [comment]
seems to have some merge issues might fix later

**Comment by chairclr** [comment]
dont know if this is ready to be merged yet

**Comment by chairclr** [comment]
might fix later

**Comment by chairclr** [comment]
this might be ready to merge, still preparing main branch

**Comment by chairclr** [comment]
guys can i get a code review

**Review by hakusaro — APPROVED** [review]

File: README.md
Comment (hakusaro) [inline README.md:46]: Same issue here

File: README.md
Comment (steviegt6) [inline README.md:49]:
```suggestion
  - Chat Message Received
```

**Review by hakusaro — CHANGES_REQUESTED** [review]

**Review by steviegt6 — CHANGES_REQUESTED** [review]

File: README.md
Comment (chairclr) [reply README.md:49]: hi tomat

**Comment by chairclr** [comment]
no dsicrption proided

**Comment by steviegt6** [comment]
> no dsicrption proided

I see!

File: README.md
Comment (steviegt6) [reply README.md:49]: hi tomat

**Comment by chairclr** [comment]
hi

**Comment by steviegt6** [comment]
LBTM (looks bood to me)

**Comment by hakusaro** [comment]
Lstm (looks Sarah to me)

**Comment by steviegt6** [comment]
LSTM (let's steal the monstitution)

**Comment by chairclr** [comment]
This PR sounds pretty cool. Any thoughts/

**Comment by hakusaro** [comment]
@CEO-Chair thoughts: revoke

**Comment by chairclr** [comment]
hi

**Comment by steviegt6** [comment]
@CEO-Chair, can we get a review before merging? LGTM

**Comment by chairclr** [comment]
> can we get a review before merging?

yeah sure

**Review by steviegt6 — COMMENTED** [review]
LGTM

**Pull request merged**
